hack/testing/logging.sh

```diff
@@ -346,6 +346,11 @@ else
        -f $OS_O_A_L_DIR/hack/templates/dev-builds.yaml \
        -v LOGGING_FORK_URL=$GIT_URL -v LOGGING_FORK_BRANCH=$GIT_BRANCH \
        | build_filter | oc create -f -"
+    
+    os::cmd::expect_success "docker pull centos:7"
+    CENTOS7_IS="{ \"kind\": \"ImageStream\",\"apiVersion\": \"v1\", \"metadata\": { \"name\": \"centos\" }, \"spec\": {\"tags\": [{\"name\":\"7\",\"from\":{\"kind\":\"DockerImage\",\"name\":\"centos/centos:7\"}}]} }"
+    os::cmd::expect_success "echo $CENTOS7_IS | oc create -n openshift -f -"
```

Comment:
i think this should be removed in favor of the changes in https://github.com/openshift/origin-aggregated-logging/pull/297